### ISSUE TITLE
API problem

### ISSUE THREAD

**Issue by itpun** [chrislessard/LocalCounselor#1]
RA seemed to have tried to develop an API for developers, but gave up on it (?):

https://stackoverflow.com/questions/34182163/how-to-get-residentadvisor-api-work

You could do some manual parsing, but could be a pain to maintain overtime. Might be worthwhile to email them and see if they're willing to give you the API access key.

**Comment by chrislessard** [comment]
Good call. For now, I was considering writing a web scraper, though I'll need to look into their policy a bit to see what they think of this.
